modules/network/src/main/java/org/apache/ignite/internal/network/serialization/ClassDescriptor.java

```diff
@@ -59,6 +65,20 @@
      */
     private final boolean isFinal;
 
+    /** Total number of bytes needed to store all primitive fields. */
+    private final int primitiveFieldsDataSize;
+    /** Total number of object (i.e. non-primitive) fields. */
```

Comment:
```suggestion
    /** Total number of object's non-primitive fields. */
```

Reply:
This changes the meaning. It was about 'non-primitive, aka reference, aka object fields'. Do you really suggest this?

Reply:
sorry, then it should be "Total number of non-primitive fields."

Reply:
Yes, this seems better. Fixed.